src/main/webapp/app/modules/upload/upload-page.tsx

```diff
@@ -109,9 +109,21 @@ export class UploadPage extends React.Component<IUploadPageProp, IUploadState> {
 
   fileValidateTypeDetectType = (source, type) =>
     new Promise((resolve, reject) => {
+      type = this.handleUndefinedFileType(source, type);
       resolve(type);
     });
 
+  handleUndefinedFileType = (source, type) => {
+    if (type === '') {
+      const re = /(?:\.([^.]+))?$/;
```

Comment:
Could you describe this regex by changing its name?

Reply:
Right, I forgot about it, thanks!